src/guessTheNumber.js

```diff
@@ -0,0 +1,20 @@
+const mysteryNumber = Math.floor(Math.random() * 100) + 1
+
+function guessTheNumber () {
+  let input = document.getElementById('answer').value
+  if((typeof input != Number) && (input.toLowerCase() == 'exit')) {
```

Comment:
This line is a bit verbose, you could drop the first check or switch it to an || instead of &&. If input === 'exit' it is safe to assume it is also not a number. This line seems to be checking if both conditions are true but you may want to switch it to if either are true.

if ( typeof input !== Number || input.toLowerCase() === 'exit' )